deeprank/generate/DataGenerator.py

```diff
@@ -802,7 +803,7 @@ def map_features(self, grid_info={},
         >>> grid_info = {
         >>>     'number_of_points' : [30,30,30],
         >>>     'resolution' : [1.,1.,1.],
-        >>>     'atomic_densities' : {'CA':3.5,'N':3.5,'O':3.5,'C':3.5},
+        >>>     'atomic_densities' : {'C':1.7, 'N':1.55, 'O':1.52, 'S':1.8},
```

Comment:
Why half of the radius ? Did I use the diameter ? There also we could use Mendeleev ...

Reply:
The Mendeleev has the same VDW values as what I'm using.